src/test/lib/sql/sql_translator_test.cpp

```diff
@@ -1586,6 +1586,107 @@ TEST_F(SQLTranslatorTest, JoinInnerComplexPredicateA) {
   EXPECT_LQP_EQ(actual_lqp, expected_lqp);
 }
 
+TEST_F(SQLTranslatorTest, JoinInnerUsingNamedColumnsSimple) {
```

Comment:
Haven't looked into the tests in detail, yet. However, the formatting looks a bit interesting. I saw that you draw inspiration from other tests (e.g., `JoinNaturalSimple`), but IMO using that much newlines is too bloaty (feel free to change it there, as well). Regarding formatting: we also usually format hand-crafted LQPs by hand and turn clang-format off.

For example, see the tests for `CastStatement`:
https://github.com/hyrise/hyrise/blob/5306d23c855bd24bc112295f582064f16a7a5a8c/src/test/lib/sql/sql_translator_test.cpp#L3225-L3245

You can also see how we test invalid user inputs (e.g., named column not present in one or both inputs) there.